src/main/java/org/openpnp/model/BoardLocation.java

```diff
@@ -68,10 +70,22 @@ public void setLocation(Location location) {
 		firePropertyChange("location", oldValue, location);
 	}
 
+	public Location getDimension() {
```

Comment:
Dimensions should be a property of Board, not BoardLocation. Board contains properties of a single PCB and since the dimensions of the PCB are fixed there is no reason to have the user have to enter this for each Job.